### DISCUSSION TITLE
Revalidate all fields in fieldArray when one of the fields is updated

### DISCUSSION THREAD

**Discussion by RuneClaeys** [react-hook-form/react-hook-form#3531] [Q&A]
I have a project where an unknown amount of rolls will need to be scanned. However each roll must have a unique barcode. When you have 2 values that are the same. The last one shows the error that this is a duplicate. but if i change the first one, the error doesn't vanish.

https://stackblitz.com/edit/react-ts-akrb3j?file=index.tsx

I tried to re-create the scenario in the above stackblits. Any help or ideas would be much appreciated!
(ps. English is not my native languate so sorry for any mistakes)

Also a visual example of what I'm after => https://imgur.com/1RMLei7

**Comment by danielbattat** [comment]
Try using trigger() in the onChange of all the fields to retrigger form validation

**Reply by bluebill1049** [reply]
Yes, for depend fields, you may want to use trigger 👍
